dts/bindings/rtc/nxp,kinetis-rtc.yaml

```diff
@@ -10,3 +11,14 @@ include: rtc.yaml
 properties:
   reg:
     required: true
+
+  clock-source:
+    type: string
+    default: "RTC"
```

Comment:
Just for simplicity, so I don't have to add this property to the dts of other platforms that contains 'nxp,kinetis-rtc', because there is no additional code operation for the driver when selecting RTC.

Reply:
if there is a default value in the binding then it needs to follow these rules

https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
Added default value description